showcase/src/assets/examples/collapsible/collapsible-custom1.html

```diff
@@ -0,0 +1,31 @@
+<stark-collapsible
```

Comment:
Can you change the name of these files so it is clear what example they are related to? `collapsible-custom1` is too ambiguous...